deploy/swarmkit/README.md

```diff
@@ -0,0 +1,57 @@
+# Run weave-socks demo on the new Docker Swarm
+
+Please refer to the [new Docker Swarm introduction](http://container-solutions.com/hail-new-docker-swarm/)
+
+# Blockers
+
+Currently, new Docker Swarm does not support running containers in privileged mode. 
+Maybe it will be allowed in the future.
+Please refer to the issue [1030](https://github.com/docker/swarmkit/issues/1030#issuecomment-232299819).
+This prevents from running Weave Scope, since it needs privileged mode.
+
+Running global plugins is not supported either.
+
+# Overview
+
+This setup includes 3 nodes for Docker Swarm.
+master1 - is the Docker Swarm manager node
+node1 and node2 - worker nodes
+
+# Pre-requisities
+
+* Vagrant
```

Comment:
Added docker-for-mac section. This will be limited to one node only, but much quicker to set up.